src/app/(proper_react)/redesign/(authenticated)/user/welcome/page.tsx

```diff
@@ -0,0 +1,58 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { getL10n } from "../../../../../functions/server/l10n";
+import { isEligible } from "../../../../../../external/onerep";
+
+export async function generateMetadata() {
+  const l10n = getL10n();
+  return {
+    title: "Welcome", // FIXME l10n.getString("breach-meta-title"),
+    twitter: {
+      card: "summary_large_image",
+      title: l10n.getString("brand-fx-monitor"),
+      description: l10n.getString("meta-desc-2"),
+      images: ["/images/og-image.webp"],
+    },
+    openGraph: {
+      title: l10n.getString("brand-fx-monitor"),
+      description: l10n.getString("meta-desc-2"),
+      siteName: l10n.getString("brand-fx-monitor"),
+      type: "website",
+      url: process.env.SERVER_URL,
+      images: ["/images/og-image.webp"],
+    },
+  };
+}
+
+export default async function UserWelcome() {
+  const eligible = await isEligible();
+
+  if (!eligible) {
+    return (
+      <main>
+        <h2>You have already used your free scan.</h2>
+      </main>
+    );
+  }
+
+  return (
+    <main>
+      <h1>Welcome to Monitor. Let’s find your exposed information.</h1>
+      <p>
+        To find and fix your exposed data, we’ll need to ask for your personal
+        info. Rest assured we will safeguard your information like it’s our own.
+      </p>
+
+      <p>
+        We believe in your right to privacy, so basic Monitor is always free.
+        Upgrade to Premium for continuous protection and automatic removal of
+        your personal info.
+      </p>
+      <a href="welcome/details">
+        <button className="button primary">Start my free scan</button>
```

Comment:
Last I checked that CSS hadn't made its way into the `(proper_react)` dir yet, but I have been keeping this branch up-to-date and it may work now, will check thanks.